test/coverage/gen_build.sh

```diff
@@ -26,6 +26,10 @@ set -e
 rm -f "${BUILD_PATH}"
 
 TARGETS=$("${BAZEL_BIN}" query ${BAZEL_QUERY_OPTIONS} "attr('tags', 'coverage_test_lib', ${REPOSITORY}//test/...)" | grep "^//")
+
+# Run the QUICHE platform api tests for coverage.
+TARGETS="$TARGETS $("${BAZEL_BIN}" query ${BAZEL_QUERY_OPTIONS} "attr('tags', 'coverage_test_lib', '@com_googlesource_quiche//:all')" | grep "^@com_googlesource_quiche")"
```

Comment:
How does this limit it to only the platform tests?

Reply:
This selects all tests in bazel/external/quiche.BUILD,  currently all tests in it are platform api tests.
There are lots of non-platform tests in QUICHE, but we are not running any of them in Envoy yet. We need to decide how to manage those tests in Envoy in the future.